experiments/Atmos_LES/bomex.jl

```diff
@@ -0,0 +1,503 @@
+#=
+@article{doi:10.1175/1520-0469(2003)60<1201:ALESIS>2.0.CO;2,
+author = {Siebesma, A. Pier and Bretherton, 
+          Christopher S. and Brown, 
+          Andrew and Chlond, 
+          Andreas and Cuxart, 
+          Joan and Duynkerke, 
+          Peter G. and Jiang, 
+          Hongli and Khairoutdinov, 
+          Marat and Lewellen, 
+          David and Moeng, 
+          Chin-Hoh and Sanchez, 
+          Enrique and Stevens, 
+          Bjorn and Stevens, 
+          David E.},
+title = {A Large Eddy Simulation Intercomparison Study of Shallow Cumulus Convection},
+journal = {Journal of the Atmospheric Sciences},
+volume = {60},
+number = {10},
+pages = {1201-1219},
+year = {2003},
+doi = {10.1175/1520-0469(2003)60<1201:ALESIS>2.0.CO;2},
+URL = {https://journals.ametsoc.org/doi/abs/10.1175/1520-0469%282003%2960%3C1201%3AALESIS%3E2.0.CO%3B2},
+eprint = {https://journals.ametsoc.org/doi/pdf/10.1175/1520-0469%282003%2960%3C1201%3AALESIS%3E2.0.CO%3B2}
+=# 
+
+using Distributions
+using Random
+using StaticArrays
+using Test
+using DocStringExtensions
+using LinearAlgebra
+
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.GenericCallbacks
+using CLIMA.Mesh.Filters
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+
+import CLIMA.DGmethods: vars_state, vars_aux
+import CLIMA.Atmos: source!, atmos_source!, altitude
+import CLIMA.DGmethods: boundary_state!
+import CLIMA.Atmos: atmos_boundary_state!, atmos_boundary_flux_diffusive!, flux_diffusive!, thermo_state
+
+# ---------------------------- Begin Boundary Conditions ----------------- #
+"""
+  BOMEX_BC <: BoundaryCondition
+  Prescribes boundary conditions for Dynamics of Marine Stratocumulus Case
+#Fields
+$(DocStringExtensions.FIELDS)
+"""
+struct BOMEX_BC{FT} <: BoundaryCondition
+  "Friction velocity"
+  u_star::FT  
+  "Sensible Heat Flux"
+  w′θ′::FT
+  "Latent Heat Flux"
+  w′qt′::FT
+end
+
+"""
+    atmos_boundary_state!(nf::Union{NumericalFluxNonDiffusive, NumericalFluxGradient},
+                          bc::BOMEX_BC, args...)
+
+For the non-diffussive and gradient terms we just use the `NoFluxBC`
```

Comment:
```suggestion
For the non-diffusive terms we use the `NoFluxBC`
```
Using no flux for "gradient fluxes" does not seem right.

Reply:
Currently based on `boundaryconditions.jl` we have a correction to the auxiliary fluxes based on the `NumericalFlux` type i.e. `NonDiffusive` or `Gradient` ; `NoFlux` is itself a misnomer I think ; More consistent names would be the `Impenetrable` , `Impermeable` , `PrescribedFlux`, `PrescribedTemperature` specifications  (or similar for Dirichlet boundaries on state variables) which Simon's #733 introduces.

Reply:
Ok. Let's get this clarified in what Simon is currently working on, and what we put underneath. This code here I do not understand, but I trust you got it right.